parm/parm_fv3diag/field_table_thompson_noaero_tke_progsigma

```diff
@@ -0,0 +1,70 @@
+# prognostic cloud water mixing ratio
+ "TRACER", "atmos_mod", "liq_wat"
+           "longname",     "cloud water mixing ratio"
+           "units",        "kg/kg"
+       "profile_type", "fixed", "surface_value=1.e30" /
+# prognostic ice water mixing ratio
+ "TRACER", "atmos_mod", "ice_wat"
+           "longname",     "cloud ice mixing ratio"
+           "units",        "kg/kg"
+       "profile_type", "fixed", "surface_value=1.e30" /
+# prognostic rain water mixing ratio
+ "TRACER", "atmos_mod", "rainwat"
+           "longname",     "rain water mixing ratio"
+           "units",        "kg/kg"
+       "profile_type", "fixed", "surface_value=1.e30" /
+# prognostic snow water mixing ratio
+ "TRACER", "atmos_mod", "snowwat"
+           "longname",     "snow water mixing ratio"
+           "units",        "kg/kg"
+       "profile_type", "fixed", "surface_value=1.e30" /
+# prognostic Grau water mixing ratio
+ "TRACER", "atmos_mod", "graupel"
+           "longname",     "graupel mixing ratio"
+           "units",        "kg/kg"
+       "profile_type", "fixed", "surface_value=1.e30" /
+# prognostic cloud water number concentration - not for non-aerosol runs
+# "TRACER", "atmos_mod", "water_nc"
+#           "longname",     "cloud  liquid water  number concentration"
+#           "units",        "/kg"
+#       "profile_type", "fixed", "surface_value=0.0" /
+# prognostic cloud ice number concentration
+ "TRACER", "atmos_mod", "ice_nc"
+           "longname",     "cloud ice water number concentration"
+           "units",        "/kg"
+       "profile_type", "fixed", "surface_value=0.0" /
+# prognostic rain number concentration
+ "TRACER", "atmos_mod", "rain_nc"
+           "longname",     "rain number concentration"
+           "units",        "/kg"
+       "profile_type", "fixed", "surface_value=0.0" /
+# prognostic ozone mixing ratio tracer
+ "TRACER", "atmos_mod", "o3mr"
+           "longname",     "ozone mixing ratio"
+           "units",        "kg/kg"
+       "profile_type", "fixed", "surface_value=1.e30" /
+# water- and ice-friendly aerosols (Thompson) - not for non-aerosol runs
+# "TRACER", "atmos_mod", "liq_aero"
+#           "longname",     "water-friendly aerosol number concentration"
+#           "units",        "/kg"
+#       "profile_type", "fixed", "surface_value=0.0" /
+# "TRACER", "atmos_mod", "ice_aero"
+#           "longname",     "ice-friendly aerosol number concentration"
+#           "units",        "/kg"
+#       "profile_type", "fixed", "surface_value=0.0" /
+# prognostic subgrid scale turbulent kinetic energy
+ "TRACER", "atmos_mod", "sgs_tke"
+           "longname",     "subgrid scale turbulent kinetic energy"
+           "units",        "m2/s2"
+       "profile_type", "fixed", "surface_value=0.0" /
+# prognotsitc sigmab tracer
+ "TRACER", "atmos_mod", "sigmab"
+           "longname",     "sigma fraction"
+           "units",        "fraction"
+       "profile_type", "fixed", "surface_value=0.0" /
```

Comment:
The ever-longer file names to just add these lines, along with needing to update multiple files that contain the same info, is a perfect example of why we need to move to a modular way of building these files. @aerorahul

Reply:
I agree. There has been quite a lot of discussion on how to build these (e.g. say what should be advected, what should be diffused vertically and so on). It would also help clean up source code as there is currently a lot of if statements regarding treatment of tracers in the interstital routines.

Reply:
Not a blocker for this PR, just more of a general ongoing issue. See also https://github.com/ufs-community/ufs-weather-model/issues/673